src/client/js/components/PageEditor/OptionsSelector.jsx

```diff
@@ -97,6 +98,17 @@ class OptionsSelector extends React.Component {
     editorContainer.saveOptsToLocalStorage();
   }
 
+  onClickFormattingMarkdownTable(event) {
+    const { editorContainer } = this.props;
+
+    const newValue = !editorContainer.state.editorOptions.formattingMarkdownTable;
+    const newOpts = Object.assign(editorContainer.state.editorOptions, { formattingMarkdownTable: newValue });
```

Comment:
Change 'formattingMarkdownTable' to 'ignoreMarkdownTableAutoFormatting'.
As a result, auto-formatting would be enabled in default.

Reply:
I think this is a good idea and user-friendly.
I will respond.